### PR TITLE
✨ Add eslint rule to enforce comment whitespacing

### PR DESCRIPTION
**Pull request by marlowpayne**
Let's enforce comment whitespacing. We already enforce this rule in ES6. As it is backwards-compatible with pre-v1 versions of eslint, let's include it there too.
## Changes
- Add `spaced-comment` eslint rule to `/javascript/.eslintrc` and `/es5/mobify-es5.yml` for enforcing (rule is already included in ES6 ruleset)
- Add comments to `/javascript/README` and `/es5/README` regarding rule enforcement
## How To Test
- Checkout code
- `npm link`
- In a project using ES5 linting, run `npm link mobify-code-style`
- Make a comment without adding whitespace to the starting comment marker (`//Bad comment`)
- `grunt lint`
- Ensure comment whitespacing is enforced
## Applicable Research Resources
- http://eslint.org/docs/rules/spaced-comment
## TODOs:
- [x] +1
- [x] Updated README
- [x] Updated CHANGELOG

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Pending Release
     - Remove `variable-for-property` rule from sass-lint
+    - Add `spaced-comment` rule enforcement to ES5
```
Comment (jeremywiebe) [inline CHANGELOG]: and "legacy" javascript?

File: es5/README.md [hunk 2]
```diff
@@ -44,6 +44,22 @@ Don't commit commented out sections of code back into the repository. Just delet
 If a piece of code is very temporarily being removed, and will be reinserted shortly, you might decide to do this anyway. Please leave a detailed comment explaining exactly why. Sorta like those post-its we leave on boxes in the fridge with our name and date so that we know when to chuck 'em out!
```
Comment (jeremywiebe) [inline es5/README.md]: 🔫  extra blank line

File: javascript/README.md [hunk 3]
```diff
@@ -46,6 +46,22 @@ Don't commit commented out sections of code back into the repository. Just delet
...
+// This is a good comment 👍
+
+/*This is
+*a bad multi-
+*line comment 👎 */
+
+/* This is a good
+* multi-line comment 👍 */
+````
+
```
Comment (jeremywiebe) [inline javascript/README.md]: 🔫  extra blank line

File: javascript/README.md [hunk 4]
```diff
@@ -46,6 +46,22 @@ Don't commit commented out sections of code back into the repository. Just delet
 If a piece of code is very temporarily being removed, and will be reinserted shortly, you might decide to do this anyway. Please leave a detailed comment explaining exactly why. Sorta like those post-its we leave on boxes in the fridge with our name and date so that we know when to chuck 'em out!
```
Comment (jeremywiebe) [inline javascript/README.md]: 🔫  extra blank line

File: es5/README.md [hunk 5]
```diff
@@ -44,6 +44,22 @@ Don't commit commented out sections of code back into the repository. Just delet
...
+
+/*This is
+*a bad multi-
+*line comment 👎 */
+
+/* This is a good
+* multi-line comment 👍 */
+````
+
+
```
Comment (jeremywiebe) [inline es5/README.md]: 🔫  extra blank line

File: javascript/README.md [hunk 6]
```diff
@@ -45,11 +45,24 @@ if (condition) {
...
+//This is a bad comment 👎
+
+// This is a good comment 👍
+
+/*This is
+*a bad multi-
+*line comment 👎 */
+
+/* This is a good
+* multi-line comment 👍 */
```
Comment (jeremywiebe) [inline javascript/README.md]: Ditto to lining up opening `*`

**Comment by jeremywiebe** [comment]
@marlowpayne I'm guessing we need this now for `es6` also, right?

File: es5/README.md [hunk 7]
```diff
@@ -42,11 +42,24 @@ if (condition) {
...
+//This is a bad comment 👎
+
+// This is a good comment 👍
+
+/*This is
+*a bad multi-
+*line comment 👎 */
+
+/* This is a good
+* multi-line comment 👍 */
```
Comment (marlowpayne) [inline es5/README.md]: LOL
All good @jeremywiebe . I much prefer that style too.

**Comment by marlowpayne** [comment]
@jeremywiebe our ES6 config already includes this rule enforcement. This PR was mainly to just back-fill it to earlier versions of JS linting.

**Comment by jeremywiebe** [comment]
@marlowpayne Oh man. I can't read!  "rule is already included in ES6 ruleset"

**Comment by jeremywiebe** [comment]
Conflict in `CHANGELOG` but otherwise 👍

**Pull request merged**